src/Transport.java

```diff
@@ -0,0 +1,29 @@
+public class Transport {
```

Comment:
```suggestion
public class Transport implements Service {
```
Тут добавим реализацию интерфейса Service